### ISSUE TITLE
Hasura MySQL Preview unable to login when using HASURA_GRAPHQL_ADMIN_SECRET

### ISSUE THREAD

**Issue by luchoster** [hasura/graphql-engine#7264]
I'm running a self hosted instance using docker, and everything works as expected when I don't define the HASURA_GRAPHQL_ADMIN_SECRET.  Adding the Admin Secret, causes the web console to just reload every time I enter the secret key. In the browser's console I get:

`{path: "$", error: "x-hasura-admin-secret/x-hasura-access-key required, but not found", code: "access-denied"}`

This is what I see after trying to login:
<img width="2135" alt="hasura-error" src="https://user-images.githubusercontent.com/694367/126360180-25d2fb8c-e911-4608-b1bd-493b2bf51df8.png">

the `docker-compose.yaml` looks like:

```
version: '3.6'
services:
  postgres:
    image: postgres:12
    restart: always
    volumes:
      - db_data:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: postgrespassword
  graphql-engine:
    image: hasura/graphql-engine:pull5655-633f084f
    ports:
      - "8080:8080"
    depends_on:
      - "postgres"
    command:
      - graphql-engine
      - --mysql-host
      - "url.com"
      - --mysql-user
      - "usr"
      - --mysql-port
      - "3306"
      - --mysql-dbname
      - "db-name"
      - --mysql-password
      - "pwd"
      - serve
    restart: always
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/postgres
      ## enable the console served by server
      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # set to "false" to disable console
      ## enable debugging mode. It is recommended to disable this in production
      HASURA_GRAPHQL_DEV_MODE: "true"
      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
      ## uncomment next line to set an admin secret
      HASURA_GRAPHQL_ADMIN_SECRET: somesecretkey
volumes:
  db_data:
```

At the moment I'm just trying to make sure I can access the web console to build the queries, but can't find anything that can help me solve this issue. According to everything that I've seen, I just have to enable the ADMIN_SECRET in the docker-compose and it should work. 

Any ideas?

Thanks.

**Comment by mattjslaton** [comment]
what version are you running and have you tried another browser?

**Comment by luchoster** [comment]
I've tried different computers (including mobile), browsers (Safari, Chrome, Brave).

In the console I get the error trying to GET `/v1alpha1/config`, so I guess V1.

Last night I was able to capture the screen that tries to load and then goes back to the login.

![Screenshot_20210719-223111~2__01.jpg](https://user-images.githubusercontent.com/694367/126369653-1abb43ec-be6a-4e71-8858-c9b502469f53.jpg)
